src/main/java/com/hillel/items_exchange/mapper/tranfer/Exist.java

```diff
@@ -0,0 +1,5 @@
+package com.hillel.items_exchange.mapper.tranfer;
+
+public interface Exist {
```

Comment:
Add javadocs here that will explain the purpose of the interface.

Reply:
Done